posit_workbench_acceptable_use_policy.md

```diff
@@ -0,0 +1,82 @@
+# Posit Workbench Acceptable Use Policy
+
+Original author(s): Terry McLaughlin
+
+Last updated: January 2023
+
+## Introduction
+
+
+
+### Purpose
+
+
+### Scope
+
+
+## Accessing Posit Workbench
+
+### Authorisation
+
+Line managers of PHS staff can authorise their requests for access to Posit Workbench.
+
+Non-PHS users must seek authorisation from ...
+
+### Changing User Access
+
+Changes to users' access can be requested by ...
+
+### User Access Tracking
+
+Public Health Scotland have a fixed number of named-user licenses for Posit Workbench, and therefore users accessing Posit Workbench will be tracked for the sole purpose of ensuring compliance with the license agreement.
+
+## Appropriate Usage
+
+### Cloud Services
+
+Posit Workbench is hosted in Microsoft Azure, a cloud computing platform.  Microsoft Azure is a pay-as-you-go platform, and therefore most activities incur cost to Public Health Scotland, particularly running a Workbench session (even when it is idle) and any data being saved out of the Microsoft Azure environment e.g. saving to a file share on the Stats server.
+
+Users must take steps to minimise these costs by ensuring that they only request the computing resources they need to carry out their work, and ensuring that their analyses are as efficient as possible at all times.
+
+### Shared Computing Resources
+
+Users must be mindful that, whilst Posit Workbench's implementation on Microsoft Azure allows for more separation between each user's Workbench sessions, the environment is still running on shared computing resources.  Therefore, users must not seek to monopolise the system for their own use, or unnecessarily overload the system by requesting excessive resources for their session.  The environment will automatically scale up and down as user demand dictates, but there is an upper limit to control overall costs to Public Health Scotland.
+
+### Personal Usage
+
+Users are prohibited from using Posit Workbench for personal projects or activities.
+
+## Auditing
+
+User activity on Posit Workbench will be routinely collated in dashboards and reports solely for the purposes of
+
+* monitoring the stability and performance of the Posit Workbench environment hosted in Microsoft Azure, and
+* calculating costs accrued by usage of Posit Workbench.
+
+Access to these audit logs, dashboards and reports will be restricted to authorised staff.  Authorisation shall be granted by ...
+
+## Training and Awareness
+
+Users can access appropriate training on the usage of Posit Workbench, R programming, GitHub and other Data Science topics related to use of Posit Workbench through the Data Science Knowledge Base.
```

Comment:
I think we should link to the Knowledge Base here for ease of access:

https://public-health-scotland.github.io/knowledge-base/

Reply:
Done.